core/app/assets/javascripts/refinery/admin.js.erb

```diff
@@ -577,13 +577,17 @@ var page_options = {
   },
 
   page_part_dialog: function(){
+    var slugOptions = {
+      separator: '_'
```

Comment:
Agree. If nothing else, Google recommends hyphens over underscores. It used to be better for SEO although I'm not sure that's still the case. 

Plus, it's marginally easier to type, matches domain restrictions, so it looks more URL-y (underscores are not allowed in domain names.)